### PR TITLE
Fix code syntax

### PR DESCRIPTION
**Pull request by thrawn01**
Fix build errors which result in code examples not highlighted properly and links not working.

### CODE REVIEW

File: source/api-webhooks-deprecated.rst [hunk 1]
```diff
@@ -131,11 +131,11 @@ one of your emails, your URL will be called with the following parameters.
  recipient             Recipient who opened.
  domain                Domain that sent the original message.
  ip                    IP address the event originated from.
- country               Two-letter `country code`_ (as specified by `ISO3166`_) the event came from or
+ country               Two-letter country code (as specified by ISO3166) the event came from or
```
Comment (highlycaffeinated) [inline source/api-webhooks-deprecated.rst]: why removing the formatting here?

File: source/api-webhooks-deprecated.rst [hunk 2]
```diff
@@ -147,9 +147,9 @@ one of your emails, your URL will be called with the following parameters.
  tag                   Message tag, if message was tagged. See :ref:`tagging`
  mailing-list          The address of mailing list the original message was sent to.
  "custom variables"    Your own custom JSON object included in the header (see :ref:`manual-customdata`).
- timestamp             Number of seconds passed since January 1, 1970 (see `securing webhooks`_).
- token                 Randomly generated string with length 50 (see `securing webhooks`_).
- signature             String with hexadecimal digits generate by HMAC algorithm (see `securing webhooks`_).
+ timestamp             Number of seconds passed since January 1, 1970
```
Comment (highlycaffeinated) [inline source/api-webhooks-deprecated.rst]: Removing the links?

File: source/api-webhooks-deprecated.rst (hunk 1 above)
Comment (thrawn01) [reply source/api-webhooks-deprecated.rst]: Those are not formatting but links to non existent link locations.

File: source/api-webhooks-deprecated.rst (hunk 2 above)
Comment (thrawn01) [reply source/api-webhooks-deprecated.rst]: Yes, don't exist anymore since being deprecated

**Pull request merged**